test/mlir/affine_conversion/core_ops.mlir

```diff
@@ -135,8 +135,8 @@ func @simple_lesseq(%arg0: !ng.tensor<2x2xf32>, %arg1: !ng.tensor<2x2xf32>) -> !
 // CHECK:       }
 // CHECK-NEXT:  }
 // CHECK:       affine.for %[[K:.*]] = 0 to 16
-// CHECK-NEXT:  affine.for {{%.*}} = 0 to 8
 // CHECK-NEXT:  affine.for %[[M:.*]] = 0 to 32
+// CHECK-NEXT:  affine.for {{%.*}} = 0 to 8
```

Comment:
Why did the loop order change ? MLIR update shouldn't cause this.

Reply:
Sorry, I think my prev build "leaked" since I didn't do make install... :(